frontend/src/components/MeetingRoom/ArchivingToggle/ArchivingToggle.tsx

```diff
@@ -78,6 +80,10 @@ const ArchivingToggle = (): ReactElement => {
               style={{ color: `${isRecording ? 'rgb(239 68 68)' : 'white'}` }}
             />
           }
+          sx={{
+            marginTop: isSmallViewport ? '0px' : '4px',
```

Comment:
Note: you'll notice that we set this here instead of `ToolbarButton`. the reason is that we want to lower only certain buttons to make sure that they align correctly with those that are displayed only on desktop

Reply:
these are all the ones that in the dropdown kebab menu but `ToolbarButton` is used across other components so it's kinda the same reason as here: https://github.com/Vonage/vonage-video-react-app/pull/95#discussion_r1972237667